hydra-node/src/Hydra/Options.hs

```diff
@@ -570,7 +570,7 @@ blockfrostQueryTimeoutParser =
     auto
     ( long "blockfrost-query-timeout"
         <> metavar "SECONDS"
-        <> value 10
+        <> value 20
```

Comment:
i think we should have a defaultBlockfrostQueryTimeout defined somewhere

Reply:
There is a value `queryTimeout` but I like your suggestion since we can reuse `defaultBFQueryTimeout` everywhere.